docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc

```diff
@@ -119,9 +119,9 @@ C:\> net start CleanSpeakWebService
 
 == Maintenance Mode
 
-You will access CleanSpeak WebService’s Maintenance Mode setup via the browser. If you installed CleanSpeak WebService on your local machine, you’d access this interface by opening http://localhost:8011/ in your browser. If CleanSpeak is running on a remote server, change the server name in the URL to match your server’s name.
+You will access CleanSpeak WebService’s Maintenance Mode setup via the browser. If you installed CleanSpeak WebService on your local machine, you’d access this interface by opening `http://localhost:8011` in your browser. If CleanSpeak is running on a remote server, change the server name in the URL to match your server’s name.
 
-The first step of Maintenance Mode will prompt you to enter your license Id. You can retrieve your license Id from the CleanSpeak website by logging into your account here: https://account.cleanspeak.com/account/
+The first step of Maintenance Mode will prompt you to enter your license Id. You can retrieve your license Id from the CleanSpeak website by logging into your account here: `https://account.cleanspeak.com/account`
```

Comment:
Since this is a real link that we expect people to be able to click on, we may not want to use back ticks. You can leave the trailing slash on this one. (see comment from above).

To test, you can just open the link in your browser and see where it ends up.